tests/testthat/setup.R

```diff
@@ -0,0 +1,41 @@
+library('data.table')
+library('qs')
+dataDir = 'data'
+params = yaml::read_yaml(file.path(dataDir, 'GSE143524.yml'))
+os = Sys.info()['sysname']
+if (os == 'Darwin') params$salmon$indexDir = gsub('/home/', '/Users/',
```

Comment:
Still not the preferred style, since indentation is so extreme. Put the code that's inside the `if` on its own line.